lib/models/locations_movies.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'locations_movies'
+});
```

Comment:
Hmmm, this brings up a good question. I'm not sure if in the real deal we'd actually make a model for the join table or not. My feeling is that we'd just rely on a helper function in either the `Location` or the `Movie` model to handle the insert via `Knex` directly.

Then again, in the API I don't think we have a model for which there is a many to many relation. Maybe when we do we'll call on your experience with `attach` for guidance 😛